src/main/java/com/google/devtools/build/lib/bazel/BazelStartupOptionsModule.java

```diff
@@ -33,10 +33,17 @@
         valueHelp = "<path>",
         help =
             "The location of the user .bazelrc file containing default values of "
-                + "Bazel options. If unspecified, Bazel uses the first .bazelrc file it finds in "
+                + "Bazel options. This option can also be specified multiple times.\n"
+                + "E.g. with "
+                + "`--bazelrc=x.rc --bazelrc=y.rc --bazelrc=/dev/null --bazelrc=z.rc`,\n"
+                + "  1) x.rc and y.rc will be read.\n"
+                + "  2) /dev/null indicates that all further `--bazelrc`s will be ignored, "
```

Comment:
This should be explained up front, explaining it inside the example makes it hard to find...

"This option can be specified multiple times. `--bazelrc=/dev/null` indicates that all further `--bazelrc` should be ignored. This is useful for disabling the search for user rc files, such as in release builds. For example: `--bazelrc=...` will use ... and ignore ...`.

Reply:
Makes sense. Corrected.